docs/Glossary.tex

```diff
@@ -47,6 +54,20 @@
   description={Ein Protokoll zur Übertragung von Daten und Steuersignalen von Industrieanlagen}
 }
 
+\newglossaryentry{OPC UA Client}
+{
+  name=OPC UA Client,
+  plural=OPC UA Clients,
+  description={Fordert bei einem \gls{OPC UA Server} Daten an und empf\"angt von diesem Alarme. Kommuniziert per \gls{OPC UA}}
+}
+
+\newglossaryentry{OPC UA Server}
+{
+  name=OPC UA Server,
+  plural=OPC UA Server,
+  description={Modelliert einen Teil einer \gls{Produktionsanlage}. Kommuniziert per \gls{OPC UA} mit \glspl{OPC UA Client}}
```

Comment:
Jetzt nochmal in ganz praezise. Es muss ja ein Server nicht alle Sensoren kennen.